tutorials/mnist_tutorial_jsma.py

```diff
@@ -143,12 +145,17 @@ def main(argv=None):
             print('Creating adv. example for target class ' + str(target))
 
             # This call runs the Jacobian-based saliency map approach
-            adv_x, res, percent_perturb = jsma(sess, x, predictions, grads,
-                                               X_test[sample_ind:
-                                                      (sample_ind+1)],
-                                               target, theta=1, gamma=0.1,
-                                               increase=True, back='tf',
-                                               clip_min=0, clip_max=1)
+            adv_x = attack.generate_numpy(
+                X=X_test[sample_ind:(sample_ind+1)],
+                target=target,
+                nb_classes=FLAGS.nb_classes,
+                sess=sess
+            )
+            # TODO: update this; how do we actually get the 'res' value? Don't want to break from
```

Comment:
we can reevaluate the predictions of the model on adversarial examples contained in the np array and check whether they are misclassified or not in the correct target class